deployer/json_parser.py

```diff
@@ -0,0 +1,38 @@
+import requests
+
+
+# Load json from a url and return this json
+def load_json_from_url(my_url):
+    r = requests.get(url=my_url)
+    print("Retrieving asgard json from: {}".format(my_url))
+    # print(r.json())
+    return r.json()
+
+
+# It checks from a json if the status is working, and return working.
+# Otherwise return the status of the stack.
+# It only check the first entry on completedTaskList section.
+def check_stack_status(json_decoded, stack_name):
+    result = None
+    print("Searching for stack: {}".format(stack_name))
+    for iterator in json_decoded["runningTaskList"]:
+        if iterator["objectId"] == stack_name:
+            result = "working"
+            break
+    if result != "working":
+        for iterator in json_decoded["completedTaskList"]:
+            if iterator["objectId"] == stack_name:
+                result = iterator["status"]
+                break
+    return result
+
+
+# Check the asgard task status by the json and return state.
+def check_asgard_stack_status(asgard_url, stack, id_task):
```

Comment:
@Arcontus I think this function doesen't do to much and have unused params, you can substitute in the place you call ir for:
```
load_json_from_url("/task/show/{0}.json".format(id_task))["status"]
``` 

and for concatenate string use the format fuinction not the +